.github/workflows/web-test.yml

```diff
@@ -19,22 +23,19 @@ jobs:
           node-version: 18
 
       - name: NPM Install
-        run: npm install
+        run: npm install -w ${{ matrix.codepath }}
```

Comment:
We should just do `npm i` for now. Later on we can distribute the dependancies so that each workspace will work by itself.